transform/publication_indexer.go

```diff
@@ -52,6 +52,10 @@ func (m *PublicationIndexer) Index(resource *models.Publication, doc solr.Docume
 		doc.Set("publisher_ssim", *resource.Publisher)
 	}
 
+	if resource.CreatedYear != 0 {
```

Comment:
What causes this to be set to `0`?

Reply:
0 is the default value for an int. So if CreatedYear isn't set, it will be 0.

(Without pointers, there is no null value. At least that is my understanding.)

Reply:
Ah, right, `0` is the "zero value" for the int type. Thanks!